Performance for Serverless on Python (#2389)
getsentry/sentry-docs: src/platforms/python/guides/gcp-functions/index.mdx

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    integrations=[GcpIntegration()],

>     traces_sample_rate=1.0, # adjust the sample rate in production as needed

Reviewer: We're moving to this language:

```suggestion
    traces_sample_rate=1.0, # We recommend adjusting this in production
});
```
